modules/shared/src/main/scala/org/tessellation/security/Hex.scala

```diff
@@ -52,6 +54,30 @@ object hex {
         }
       } yield pk
 
+    def toPublicKeyByEC[F[_]: Async: SecurityProvider]: F[PublicKey] =
```

Comment:
Is it going to be a different PublicKey than the one returned by `toPublicKey`?

Reply:
It's encoded differently to hex, our method for decoding doesn't work (uses different Spec - X509 vs EC)

Reply:
Ok, thanks for the explanation